src/gnn_reco/models/task/reconstruction.py

```diff
@@ -75,3 +75,19 @@ def _forward(self, x):
         log_var = x[:,1]
         pred = torch.stack((energy, log_var), dim=1)
         return pred
+
+class VertexReconstruction(Task):
+    # Requires four features, x, y, z and t
+    nb_inputs = 4
+
+    def _forward(self, x):
+        # Initally do nothing to the output
+        return x.unsqueeze(1) 
+
+class PositionReconstruction(Task):
+    # Requires three features, x, y, z
+    nb_inputs = 3
+
+    def _forward(self, x):
+        # Initally do nothing to the output
+        return x.unsqueeze(1) 
```

Comment:
Same as first comment above.

Reply:
At some point (not for this PR, necessarily) it might be beneficial to use this task (position) as part of the vertex reconstruction task above, since the former is essential a sub-task of the latter.